src/components/Collaborative/utils/getEditorSources.ts

```diff
@@ -0,0 +1,19 @@
+import md5 from "md5";
+
+export default function getEditorSources(content) {
+    let sourceReference = 0;
+    return content.JSON.content
```

Comment:
suggestion: pass `content.JSON.content` to the function instead of looking at these properties inside the function.